api/proxy/test/testutils/setup.go

```diff
@@ -114,7 +145,7 @@ func ParseBackend(inputString string) (Backend, error) {
 func GetBackend() Backend {
 	backend, err := ParseBackend(os.Getenv(backendEnvVar))
 	if err != nil {
-		panic(fmt.Sprintf("BACKEND must be = memstore|testnet|sepolia|preprod. parse backend error: %v", err))
+		panic(fmt.Sprintf("BACKEND must be = memstore|hoodi|sepolia. parse backend error: %v", err))
```

Comment:
This panic message says "BACKEND must be =..." but then lists out strings that don't precisely match available options